components/clickup/clickup.app.js

```diff
@@ -0,0 +1,205 @@
+      label: "Workspace",
+      async options() {
+        const workspaces = (await this.getWorkspaces()).teams;
+        return workspaces.map((workspace) => {
+          return {
+            label: workspace.name,
+            value: workspace.id,
+          };
+        });
+      },
+    },
+    space: {
+      type: "string",
+      label: "Space",
+      async options({ workspace }) {
+        const spaces = (await this.getSpaces(workspace)).spaces;
+        return spaces.map((space) => {
+          return {
+            label: space.name,
+            value: space.id,
+          };
+        });
+      },
+    },
+    folder: {
+      type: "string",
+      label: "Folder",
+      async options({ space }) {
+        const folders = (await this.getFolders(space)).folders;
+        return folders.map((folder) => {
+          return {
+            label: folder.name,
+            value: folder.id,
+          };
+        });
+      },
+      optional: true,
+    },
+    list: {
+      type: "string",
+      label: "List",
+      async options({
+        folder, space,
+      }) {
+        const lists = folder
+          ? (await this.getLists(folder)).lists
+          : (await this.getFolderlessLists(space)).lists;
+        return lists.map((list) => {
+          return {
+            label: list.name,
+            value: list.id,
+          };
+        });
+      },
+    },
+    assignees: {
+      type: "string[]",
+      label: "Assignees",
+      description: "Select the assignees for the task",
+      async options({ workspace }) {
+        const members = await this.getWorkspaceMembers(workspace);
+        return members.map((member) => {
+          return {
+            label: member.user.username,
+            value: member.user.id,
+          };
+        });
+      },
+      optional: true,
+    },
+    tags: {
+      type: "string[]",
+      label: "Tags",
+      description:
+        "Select the tags for the task to filter when searching for the tasks",
+      async options({ space }) {
+        const tags = (await this.getTags(space)).tags;
+        return tags.map((tag) => {
+          return tag.name;
+        });
+      },
+      optional: true,
+    },
+    status: {
+      type: "string",
+      label: "Status",
+      description: "Select the status of the task",
+      async options({ list }) {
+        const statuses = (await this.getList(list)).statuses;
+        return statuses.map((status) => {
+          return status.status;
+        });
+      },
+      optional: true,
+    },
+    task: {
+      type: "string",
+      label: "Task",
+      async options({
+        list, page,
+      }) {
+        const tasks = (await this.getTasks(list, page)).tasks;
+        return tasks.map((task) => {
+          return {
+            label: task.name,
+            value: task.id,
+          };
+        });
+      },
+    },
+    priority: {
+      type: "integer",
+      label: "Priority",
+      description: `1 is Urgent
```

Comment:
@dylburger will line breaks be preserved in the description?

Reply:
Looks like line breaks are preserved in the description in the UI.
<img width="315" alt="Screen Shot 2021-08-16 at 3 54 42 PM" src="https://user-images.githubusercontent.com/10889229/129639156-f9665432-be9b-4f9f-8562-770ea3e016c1.png">